app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.example.drawingactivity">
 
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Where is the internet permission?

Reply:
Removed this, I had thought I would need it when I tested on my actual phone but I don't think that was the case.